Content.Client/Medical/Cryogenics/CryoPodSystem.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Content.Shared.Medical.Cryogenics;
+using Robust.Client.GameObjects;
+using DrawDepth = Content.Shared.DrawDepth.DrawDepth;
+
+namespace Content.Client.Medical.Cryogenics;
+
+public sealed class CryoPodSystem: VisualizerSystem<CryoPodVisualsComponent>
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<InsideCryoPodComponent, ComponentAdd>(OnCryoPodInsertion);
```

Comment:
I think the ComponentAdd event runs when the components are added to the entity, which means it doesn't guarantee that the SpriteComponent has been added yet.

I think MapInitEvent is better instead, which guarantees that all components have been added AND initialized.

Reply:
It's pre-serialization data and unless you really need it (outside of engine you likely don't) you're better off using startup.

Reply:
I've changed it to ComponentStartup, thank you!